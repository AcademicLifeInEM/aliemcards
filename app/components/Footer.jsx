import React from 'react';
import { Link } from 'react-router';
import CreativeCommonsIcon from 'svg-react-loader?name=CreativeCommonsIcon!../assets/images/by-nc-nd.svg'; // eslint-disable-line

export default () => (
  <div className="footer" role="contentinfo">
    <div className="row footer__cc">
      <div className="footer__cc-image">
        <a rel="license" href="http://creativecommons.org/licenses/by-nc-nd/3.0/">
          <CreativeCommonsIcon />
        </a>
      </div>
      <div className="footer__cc-text">
        This work is licensed under a
        <a rel="license" href="http://creativecommons.org/licenses/by-nc-nd/3.0/">
          {' '}Creative Commons Attribution-NonCommercial-NoDerivs 3.0 Unported License
        </a>.
      </div>
    </div>
    <div className="row footer__links">
      <div><Link to="/contact">Questions or Suggestions?</Link></div>
      <div><Link to="/contact">Get Involved</Link></div>
      <div><Link to="/about">About</Link></div>
    </div>
  </div>
);
