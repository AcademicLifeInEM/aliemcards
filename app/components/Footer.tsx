import * as React from 'react';
import { Link } from 'react-router-dom';
import { CCIcon } from '../utils/Shims';

export default () => (
  <div className="footer" role="contentinfo">
    <div className="row footer__cc">
      <div className="footer__cc-image">
        <a rel="license" href="http://creativecommons.org/licenses/by-nc-nd/3.0/">
          <CCIcon />
        </a>
      </div>
      <div className="footer__cc-text">
        This work is licensed under a{' '}
        <a rel="license" href="http://creativecommons.org/licenses/by-nc-nd/3.0/">
          Creative Commons Attribution-NonCommercial-NoDerivs 3.0 Unported License
        </a>.
      </div>
    </div>
    <div className="row footer__links">
      <div><Link to="/contact">Contact</Link></div>
      <div><Link to="/contact">Get Involved</Link></div>
      <div><Link to="/about">About</Link></div>
    </div>
  </div>
);
