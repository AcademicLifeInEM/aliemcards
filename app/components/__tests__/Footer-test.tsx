jest.mock('react-router-dom');

import * as React from 'react';
import * as renderer from 'react-test-renderer';
import Footer from '../Footer';

describe('<Footer />', () => {
  it('should render correctly', () => {
    const component = renderer.create(
      <Footer />,
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
