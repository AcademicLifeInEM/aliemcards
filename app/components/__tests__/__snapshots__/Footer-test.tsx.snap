// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should render correctly 1`] = `
<div
  className="footer"
  role="contentinfo"
>
  <div
    className="row footer__cc"
  >
    <div
      className="footer__cc-image"
    />
    <div
      className="footer__cc-text"
    >
      This work is licensed under a
       
      <a
        href="http://creativecommons.org/licenses/by-nc-nd/3.0/"
        rel="license"
      >
        Creative Commons Attribution-NonCommercial-NoDerivs 3.0 Unported License
      </a>
      .
    </div>
  </div>
  <div
    className="row footer__links"
  >
    <div />
    <div />
    <div />
  </div>
  <div
    className="row row--centered"
  >
    <div
      className="container footer__disclaimer"
    >
      ALiEM Cards content is intended for use by healthcare professionals. Users of this site should exercise their own clinical judgment as to the reliability and accuracy of its information. Non-healthcare-professionals who use this site do so at their own risk. Content on this site should not be construed as medical advice. Users should always confer with medical profoessionals regarding decisions about medications, evaluations and other medical treatments.
    </div>
  </div>
</div>
`;
